packages/docs/celo-codebase/protocol/proof-of-stake/locked-gold-rewards.md

```diff
@@ -6,7 +6,17 @@ Rewards to Locked Gold are totally independent from validator and validator grou
 
 ![](https://storage.googleapis.com/celo-website/docs/locked-gold-rewards.jpg)
 
-First, an on-target reward rate is determined. The protocol has a target for the proportion of circulating CELO that is locked and used for voting, and adjusts the reward rate to increase or reduce the attractiveness of locking up additional supply. This aims to balance having sufficient liquidity for CELO, while making it more challenging to buy enough CELO to meaningfully influence the outcome of a validator election.
+## Adjusting the Reward Rate for Voting Participation
+
+The protocol has a target for the proportion of circulating CELO that is locked and used for voting. An on-target reward rate is determined and then adjusted at every epoch to increase or reduce the attractiveness of locking up additional supply. This aims to balance having sufficient liquidity for CELO, while making it more challenging to buy enough CELO to meaningfully influence the outcome of a validator election.
+
+The reward rate is adjusted as follows: 
+
+![](https://storage.googleapis.com/celo-website/docs/voting_reward_rate_adjustment_equation.png)
```

Comment:
As discussed, it'd be nicer if this were a client rendered latex formula, but I think it's OK to have the PNG file if it can't be done.